sunpy/map/sources/sdo.py

```diff
@@ -130,6 +130,16 @@ def __init__(self, data, header, **kwargs):
                 self._set_symmetric_vmin_vmax()
         self._nickname = self.detector
 
+    @property
+    def waveunit(self):
+        """
+        The `~astropy.units.Unit` of the wavelength of this observation.
+
+        Most HMI files seem to not have a parseable WAVEUNIT key so if it cannot be found
```

Comment:
Do they lack the keyword or is it just gibberish to astropy.units?

Reply:
They lack the keyword (or any indication as to what the unit might be, we try to parse WAVEUNIT *many* ways in `io.fits`) so `super().waveunit` returns `None`.